### PR TITLE
add code quotes where logical, and remove some () after methods

### LINKED ISSUE #70: Methods described in content sections shouldn't have () after them, and methods should consistently be in code

### PR DESCRIPTION
**Pull request by rdahlman3**
Add code quotes around sections where we're talking about methods, or the actual C# entities, or properties
Remove some parentheses after methods in the code content

Closes #70

### CODE REVIEW

File: content/efcore/getting-started/loading-related-data.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ using (var context = new LibraryContext())
 :::repl{data-name=field-not-included} 
 :::
  
-We need to explicitly tell EF Core if we want to load a non-primitive type like the Author field. We do this using the `Include` method. In the example below, the book that is returned will include its author's information. 
+We need to explicitly tell EF Core if we want to load a non-primitive type like the `Author` field. We do this using the `Include` method. In the example below, the book that is returned will include its author's information.
```
Comment (arynhuck) [inline content/efcore/getting-started/loading-related-data.md]: Change "its author's information" to "its author information"

File: content/efcore/getting-started/querying.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ using (var context = new LibraryContext())
  
 In order to interact with the database via EF Core, we must first create an instance of our context (`LibraryContext`). `LibraryContext` extends EF Core's [`DbContext`](https://msdn.microsoft.com/library/system.data.entity.dbcontext) class which manages entity objects during runtime, and allows us to perform common database operations like running queries.
 
-Notice that we create the context with the `using` keyword. This automatically disposes the context after the using block has finished executing. Alternatively, we could manually call `LibraryContext.dispose()`, but the `using` method is more convenient and readable. It is imperative that we dispose of the context after we are finished using it because it holds an open connection to the database. 
+Notice that we create the context with the `using` keyword. This automatically disposes the context after the using block has finished executing. Alternatively, we could manually call the `dispose` method on our `LibraryContext`, but the `using` method is more convenient and readable. It is imperative that we dispose of the context after we are finished using it because it holds an open connection to the database.
```
Comment (arynhuck) [inline content/efcore/getting-started/querying.md]: nah

File: content/efcore/getting-started/understanding-data-models.md [hunk 3]
```diff
@@ -43,6 +43,6 @@ public class LibraryContext : DbContext
...
+When mapping the data model to the database, EF Core makes some assumptions which allow it to perform the mapping automatically. For example, it expects each entity in the data model to correspond to a database table that shares the same name as the `DbSet<>` property in the `DbContext`. In our library example, a matching database would have a `Books` table and a `Readers` table. Likewise, it expects each property of an entity to correspond to a table column that shares the same name and datatype. Thus, the `Books` table in our example would have an `Id` column with a `INT` datatype, a `Title` column with a `VARCHAR` datatype, and a `PublicationYear` column with an `INT` datatype. These assumptions made by EF Core are referred to as **conventions**, and they are simply its default behavior.
```
Comment (arynhuck) [inline content/efcore/getting-started/understanding-data-models.md]: Table and column names get ""

**Review by helmoski — APPROVED** [review]

**Comment by arynhuck** [comment]
Change the things and it'll be 🌮

File: content/efcore/getting-started/loading-related-data.md (hunk 1 above)
Comment (arynhuck) [reply content/efcore/getting-started/loading-related-data.md]: change to `code` and cut information

**Review by arynhuck — APPROVED** [review]
🌮 :shipit:

**Pull request merged**